luvem/http.lua

```diff
@@ -169,6 +169,7 @@ function ServerResponse:finish(chunk)
   local function maybeClose()
     self:emit('finish')
     self.socket:_end()
+    collectgarbage()
```

Comment:
without this explicit call, the memory leak persists. Moreover, the docs suggests:
> Keep amount of allocated objects small and run the GC after every iteration of the main loop
http://lua-users.org/wiki/LuaInRealTimePrograms